games-strategy/ufo-ai/ufo-ai-2.5.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils flag-o-matic games
+
+MY_P=${P/ufo-ai/ufoai}
+
+DESCRIPTION="UFO: Alien Invasion - X-COM inspired strategy game"
+HOMEPAGE="http://ufoai.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ufoai/${MY_P}-source.tar.bz2
+	mirror://sourceforge/ufoai/${MY_P}-data.tar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug server editor +client sse profile"
+
+# Dependencies and more instructions can be found here:
+# http://ufoai.ninex.info/wiki/index.php/Compile_for_Linux
+DEPEND="!server? (
+		virtual/opengl
+		virtual/glu
+		media-libs/libsdl2
+		media-libs/sdl2-image[jpeg,png]
+		media-libs/sdl2-ttf
+		media-libs/sdl2-mixer
+		virtual/jpeg
+		media-libs/libpng:0
+		media-libs/libogg
+		media-libs/libvorbis
+		x11-proto/xf86vidmodeproto
+	)
+	net-misc/curl
+	sys-devel/gettext
+	sys-libs/zlib
+	editor? (
+		dev-libs/libxml2
+		virtual/jpeg
+		media-libs/openal
+		x11-libs/gtkglext
+		x11-libs/gtksourceview:2.0
+	)"
+RDEPEND="${DEPEND}"
+
+S=${WORKDIR}/${MY_P}-source
+
+pkg_setup() {
+	if use profile; then
+		ewarn "USE=\"profile\" is incompatible with the hardened profile's -pie flag."
+	fi
```

Comment:
you walked into a trap and missed `games_pkg_setup` ;)
but as said... the eclass is deprecated anyway